src/main/java/frc/robot/Robot.java

```diff
@@ -177,6 +176,49 @@ public void robotPeriodic() {
             SmartDashboard.putNumber("ClosestStandRow", -1);
         }
 
+        Command collectFromSourceNotAuto = new ParallelCommandGroup(
+                new ArmTelescopicMoveToLength(armTelescopicSystem,
+                        Math.sqrt(Math.pow(RobotMap.SWERVE_DISTANCE_FROM_SOURCE,2) + Math.pow(RobotMap.SOURCE_HEIGHT-RobotMap.SWERVE_HEIGHT,2))),
+                Commands.runOnce(()->  armJointControlCommand.setTargetPosition(Math.atan(RobotMap.SOURCE_HEIGHT-RobotMap.SWERVE_HEIGHT/RobotMap.SWERVE_DISTANCE_FROM_CORAL))),
```

Comment:
why the calculation? it will be a constant angle, just need to check on the field what. Which will be done after the field and robot are complete.

The same is true for all these calculations done in other groups